release_creation/snapshot/install.sh

```diff
@@ -1,7 +1,7 @@
 set -e
+export DBT_PSYCOPG2_NAME=psycopg2
 rm -rf target
-mkdir target
-
+mkdir target 
 pip install -r $1/requirements/$2.requirements.txt \
```

Comment:
Can we get names on these? Also, make sure to double quote the argument because there are any spaces, this will explode big time.

Reply:
added!